scripts/install-cni.sh

```diff
@@ -145,7 +145,9 @@ if [ "$ENABLE_MASQUERADE" == "true" ]; then
   fi
 fi
 
-if [ "${ENABLE_PRIVATE_IPV6_ACCESS:-}" == "true" ] || [ "$ENABLE_IPV6" == "true" ]; then
+STACK_TYPE=$(printf '%s' "$response" | jq '.metadata.labels."cloud.google.com/gke-stack-type"')
+
+if [ "${ENABLE_PRIVATE_IPV6_ACCESS:-}" == "true" ] || [[ "${STACK_TYPE:-}" == "IPV4_IPV6" ] || [[ -z "$STACK_TYPE" && "$ENABLE_IPV6" == "true" ]]; then
```

Comment:
`"${STACK_TYPE:-}" == '"IPV4_IPV6"'` note the quotes.

Reply:
Fixed.